scripts/uosc.lua

```diff
@@ -247,6 +247,7 @@ local function create_default_menu()
 		{title = 'Open file', value = 'script-binding uosc/open-file'},
 		{title = 'Playlist', value = 'script-binding uosc/playlist'},
 		{title = 'Chapters', value = 'script-binding uosc/chapters'},
+		{title = 'Editions', value = 'script-binding uosc/editions'},
```

Comment:
Move this under `Utils`, below `Audio devices`. None is using editions, and they are not available in 99+% of files, so this shouldn't be in the main menu for most common features.